contributors/devel/help-wanted.md

```diff
@@ -2,12 +2,91 @@
 
 Items marked with the `help wanted` label need to ensure that they are:
 
-- Sufficiently actionable: clear description of what should be done
-- Tractable for new/casual contributors: there is documentation how that type of change should be made
-- Goldilocks priority: Not too high that a core contributor should do it, but not too low that it isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
-- Up to date: Often these issues become obsolete and have already been done, are no longer desirable, no longer make sense, change in priority, change in difficulty, etc.
+- **Low Barrier to Entry**<br/>
+  It should be tractable for casual contributors. Documentation on how that type of change should be made should already exist.
+
+- **Clear Task**<br/>
+  The task is agreed upon and does not require further discussions
+  in the community. Call out if that area of code is untested and requires new fixtures.
+
+  API / CLI behavior is decided and included in the OP issue, for example: _"The new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`"_, with expected validations called out.
+
+- **Goldilocks priority**<br/>
+  Not too high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
+
+- **Up-To-Date**<br/>
+  Often these issues become obsolete and have already been done, are no longer desired,
+  no longer make sense, have changed priority or difficulty , etc.
 
 Related commands:
 
 - `/help` : adds the `help wanted` label to an issue
-- `/remove-help` : removes the `help wanted` label from an issue
\ No newline at end of file
+- `/remove-help` : removes the `help wanted` label from an issue
+
+# Good First Issue
+
+Items marked with the `good first issue` label are intended for _new contributors_.
+These make it clear to new contributors that they are welcome, valued, and provide
+a clear starting point for joining the project. After a contributor has successfully
+completed 1-2 `good first issue`'s, they should be ready to move on to `help wanted` items, saving remaining `good first issue`'s for other new contributors.
```

Comment:
Could we add a sentence to encourage experienced contributors to do this in the spirit of growing our community? My worry is that folks will see this and feel like they don't have the bandwith to provide assistance.